src/components/table-of-contents-controller/table-of-contents-controller.tsx

```diff
@@ -19,9 +19,9 @@ export class TableOfContentsController {
   @Prop() titleFilterPlaceholder: string = 'Search the table of contents';
 
   /**
-   * If true, shows clear title filter button
+   * If true, hides clear title filter button
    */
-  @Prop() titleFilterClearable: boolean = true;
+  @Prop() hideClearTitleFilterButton: boolean = false;
```

Comment:
Minor: FWIW, `titleFilterClearable` is a pretty decent pattern (`...able` to indicate a type of behaviour). Vuetify uses it.

Reply:
I agree. however I thought `hideClearTitleFilterButton` would be nice because using it would mean that user simply has to add the attribute
i.e
```
<component hide-clear-title-filter></component>
```

whereas if we were to use `titleFilterClearable`
in a static HTML file, should a user need to hide it, they would have to go
```
<component title-filter-clearable="false"></component>
```
do you still want to get with `titleFilterClearable `. ?
I don't mind changing it, but that's just my reasoning behind going that way

Reply:
Ah, good point, it's about being able to change the setting. Let's go with what you have. The alternative is to allow boolean "yes/no" or "true/false" or whatever for values but I think your solution is clearer.